From: Luca Bruno <lucab@debian.org>
From: Ximin Luo <infinity0@debian.org>
Description: Remove stage1 extra dependencies (cargo 0.17)
 Fetching extra dev-dependencies is not yet supported by bootstrap.py
 and they are not need to build. However, cargo will try to download
 them before building stage1.
 .
 For patches for earlier versions of cargo, see the git history for this file.
Forwarded: not-needed
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -47,8 +47,8 @@
 
-[target.'cfg(windows)'.dependencies]
-advapi32-sys = "0.2"
-kernel32-sys = "0.2"
-miow = "0.1"
-psapi-sys = "0.1"
-winapi = "0.2"
+#[target.'cfg(windows)'.dependencies]
+#advapi32-sys = "0.2"
+#kernel32-sys = "0.2"
+#miow = "0.1"
+#psapi-sys = "0.1"
+#winapi = "0.2"
 
